src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseResultTestService.java

```diff
@@ -411,10 +401,11 @@ public void shouldCreateResultOnCustomDefaultBranch(String defaultBranch, Object
         programmingExerciseStudentParticipation.setProgrammingExercise(programmingExercise);
 
         final var result = gradingService.processNewProgrammingExerciseResult(solutionParticipation, resultNotification);
-        assertThat(result).isPresent();
+        assertThat(result).isNotNull();
     }
 
     // Test
+    // TODO: why is this test not invoked at all?
```

Comment:
TODO left on purpose?

Reply:
This was one of the disabled tests that #6845 removed, seems like this method was leftover in that PR.